pulp_container/app/tasks/builder.py

```diff
@@ -68,15 +73,13 @@ def add_image_from_directory_to_repository(path, repository, tag):
     manifest_digest = calculate_digest(bytes_data)
     manifest_text_data = bytes_data.decode("utf-8")
 
-    manifest = Manifest(
+    manifest, _ = Manifest.objects.update_or_create(
```

Comment:
Now this needs some explanation. Also i think we cannot do this. Content is immutable by design. (That is like a result of the number one Pulp architecture decisions that content is supposed to be shared.)
Did you mean `get_or_create`?

Reply:
Thank you for the review and explaining about `Content` immutability (I didn't know that)!
I was trying to fix an issue, [found by Lubos++](https://github.com/pulp/pulp_container/pull/1687#pullrequestreview-2240031768), where trying to reuse a Containerfile in different build tasks would fail.
Since `Content` must be immutable, I agree that `get_or_create` should be used in these cases.